instrumentation-appender-api/src/main/java/io/opentelemetry/instrumentation/appender/api/SdkLogEmitter.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.appender.api;
+
+final class SdkLogEmitter implements LogEmitter {
```

Comment:
`DelegatingLogEmitter`?

Reply:
Same advice applies to `SdkLogBuilder`, `SdkLogEmitterBuilder`, `SdkLogEmitterProvider`.